kernel/src/scan/data_skipping.rs

```diff
@@ -97,8 +151,8 @@ fn as_data_skipping_predicate(expr: &Expr) -> Option<Expr> {
             op: UnaryOperator::Not,
```

Comment:
qq: if we `use UnaryOperator::*` would that allow `UnaryOperation { op: Not, expr } =>{` on one line? 
Or does rustfmt still force it to multiple lines just because we constrained the type?